.gitignore

```diff
@@ -181,6 +181,7 @@ synfig-studio/po/synfigstudio.pot
 synfig-studio/po/stamp-it
 synfig-studio/src/gui/synfigstudio
 synfig-studio/synfigstudio-thumbnailer.schemas
+synfig-studio/tmpwrk3127/*
```

Comment:
I have noticed "tmpwrk" dir uses random numbers at the end each time. So, I suggest to change it to 
```synfig-studio/tmpwrk*```

Reply:
done changes